### ISSUE TITLE
Closeing nvda and opening system access causes skype 5 to crash

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1010]
**Reported by mike.reiser on 2010-11-01 06:24**
With skype 5, closeing nvda and starting System Access causes Skype to either hang and stop responding, or produce the standard "The instruction at xxxxxx can not be read" message.  STR:
1.  With Skype running, close nvda and either bring up System Access if you have it or load http://www.satogo.com.  
2.  Log in and load System Access.  

Shortly after System Access loads, skype crashes.  This doesn't seem to happen when switching from System Access to nvda, which seems to make me think that something with nvda might be causeing the issue.  Expected results: Skype should behave normally.  

Actual results: Skype exhibits the behavior noted above.

**Comment by dkager** [comment]
Can't reproduce with Skype 7 and SAToGo.
